packages/internal/src/index.js

```diff
@@ -4,6 +4,7 @@
 
 export * from './config.js';
 export * from './debug.js';
+export * from './errors.js';
```

Comment:
Aside, this “barrel module” is a likely source of “bundle bloat” because it imports facilities the importer may not be using anywhere in their contract.

Reply:
The only thing left in internal/src/errors.js is `NonNullish`, so I don't think we need to worry too much about this one specifically.

Resolving.